libbeat/docs/release-notes/highlights/highlights-7.6.0.asciidoc

```diff
@@ -18,8 +18,58 @@ For a list of bug fixes and other changes, see the {beats}
 
 // tag::notable-highlights[]
 
-//[float]
-//==== Highlight
+[float]
+==== Support added for TLS 1.3
+
+In 7.6, we've added support for TLS 1.3, enabling you to take advantage
+of the security and performance improvements available in TLS 1.3 when {beats}
+connects to an {es} cluster that supports it.
+
+//REVIEWERS: Any more details to add here. GitHub issues/PRs were spare.
```

Comment:
That's it basically. Default enabled versions will be 1.1, 1.2, and 1.3. Older Beats used to enable 1.0, 1.1, 1.2 by default.

In general TLS 1.0 and 1.1 are not deemed secure enough anymore.